src/MassTransit.RabbitMqTransport/RabbitMQRoutingkeyFormatter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace MassTransit.RabbitMqTransport
+{
+    public class RabbitMqRoutingkeyFormatter: IRoutingkeyFormatter
+    {
+        public string createRoutingkeyForType(Type messageType)
+        {
+            return "";
```

Comment:
No it should not.
Masstransit will fail to create the correct bindings when "NULL" is returned.
Since the RabbitMq.Client library does not accept "NULL" references as routingkey.